06.bowling_object/frame.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require_relative 'shot'
+
+class Frame
+  attr_reader :shots, :frame_number
```

Comment:
```suggestion
  attr_reader :shots
```

frame_numberは外部に公開しなくてもよいのでは